app/ide-desktop/lib/content/src/index.ts

```diff
@@ -37,7 +38,7 @@ const FETCH_TIMEOUT = 300
 // === Live reload ===
 // ===================
 
-if (IS_DEV_MODE) {
+if (IS_DEV_MODE && !detect.isRunningInElectron()) {
```

Comment:
Why are we disabling live reload when running in Electron?

Reply:
Per @somebody1234:
> why we are disabling live reload in electron: live reload depends on esbuild server, electron app doesn't run the esbuild server (main issue is that the ports conflict). it might be possible to run esbuild from inside electron, it might be a good idea to open an issue on the cloud-v2 repo for that?
>
> yeah, i dont think it's worked for a while - not since i switched it to esbuild at least
note that devmode was previously false for ide watch: https://github.com/enso-org/enso/pull/7022/files#diff-630face3223d6e85ea83529c29722ede4d3573044b4fd33590a778b26b455346R100
> so the live reload wouldn't have been enabled anyway

Reply:
crosspost from discord: live reload relies on the `/esbuild` endpoint from esbuild, which does not exist when running the electron app - note that this PR also changes `IS_DEV_MODE` from `false` to `true` for ide watch